coalition/index.html

```diff
@@ -87,7 +87,7 @@
 		</div>
 		</nav>
 
-		<header data-aos="fade-up" style="background-image: url('../assets/images/coalition.png'); background-size: cover; background-repeat: no-repeat; min-height: 120vh;background-position:top;">
```

Comment:
Should try to refactor all these inline styles at some point...

Reply:
Yes, we should move them under the `headers` tag in `main.scss` if they all have the same styling and just specify the image url here.